apps/api/src/app/analytics/analytics.controller.ts

```diff
@@ -26,4 +31,41 @@ export class AnalyticsController {
       success: true,
     };
   }
+
+  @Post('/identify')
+  @ExternalApiAccessible()
+  @UserAuthentication()
+  async identifyUser(@Body() userData: any, @UserSession() user: UserSessionData): Promise<any> {
+    try {
+      const hubspotPortalId = '44416662';
+      const hubspotFormId = 'fc39aa98-4285-4322-9514-52da978baae8';
```

Comment:
Left them here hardcoded, as those are public values.

Reply:
Do those identifiers tend to change? if there are, should we consider storing them in the environment variables?